### DISCUSSION TITLE
[Campaign] Opening Assistant's skill tree crashes the game

### DISCUSSION THREAD

**Discussion by Vulpeseptem** [FakeFishGames/Barotrauma#14972] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

In solo campaign when opening skill tree in my assistant my game crashes.
I tried Loading earlier save in station and it worked normally, was able to open assistant's skill tree.
However when i start mission and try to open assistant's skill tree game still crashes.
Other characters dont have that issue.

### Reproduction steps

1. Open game
2. load save
3. try opening assistants skills
4. crash

Or 
1. open game
2. load earlier save
3. start mission
4. try opening assistants skills
5. crash

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

[crashreport (2).log](https://github.com/user-attachments/files/17533722/crashreport.2.log)
[crashreport (3).log](https://github.com/user-attachments/files/17533723/crashreport.3.log)
[crashreport (4).log](https://github.com/user-attachments/files/17533724/crashreport.4.log)
[crashreport (5).log](https://github.com/user-attachments/files/17533725/crashreport.5.log)

**Comment by Regalis11** [comment]
Thank you for the report!
We have an existing report about this here: https://github.com/FakeFishGames/Barotrauma/discussions/14940. It seems that this issue has to do with pets (or defense bots) in your team: it should be possible to temporarily work around it by not keeping any pets or bots around. We've implemented a fix for this in our private repo, but it will unfortunately take a few more days before we're able to push it out into the public version. The fix is available already in the unstable branch though, so switching to that would also allow you to work around the issue for now.

Sorry for the inconvenience! Closing as a duplicate of https://github.com/FakeFishGames/Barotrauma/discussions/14940.
